updateversion.sh

```diff
@@ -47,9 +53,9 @@ update_sphinx_config () {
       -i $1
 }
 
-update_config () {
+update_app () {
```

Comment:
Are there future plans to extend this function with other updates? If not I would name it `update_app_version`...